device/src/region/mod.rs

```diff
@@ -178,7 +178,16 @@ impl Configuration {
         mut_region_dispatch!(self, get_data_frequency, random)
     }
     pub(crate) fn get_rx_delay(&self, frame: &Frame, window: &Window) -> u32 {
-        region_dispatch!(self, get_rx_delay, frame, window)
+        match frame {
+            Frame::Join => match window {
+                Window::_1 => JOIN_ACCEPT_DELAY1,
+                Window::_2 => JOIN_ACCEPT_DELAY2,
+            },
+            Frame::Data => match window {
+                Window::_1 => self.receive_delay1,
+                Window::_2 => RECEIVE_DELAY2,
```

Comment:
It appears here that RECEIVE_DELAY2 could be changed separately from receive_delay1. My reading of the 1.0.3 specification, section 5.7 `The second reception slot opens one second after the first reception slot.` suggest otherwise. Did I miss something? :slightly_smiling_face:

Reply:
Good point, I think we should add the increment as a constant instead and always add that. (And not make RECEIVE_DELAY2 configurable)